<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCommandLinkButton" name="btn_Publish">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="btn_Subcribe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Subcribe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="btn_reconnect">
        <property name="text">
         <string>ReConnect</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_QoSTitle">
          <property name="text">
           <string>QoS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_QoS">
          <property name="text">
           <string>QoS Value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_subscribr_status">
        <property name="text">
         <string>Subcribe Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_MqttStateTitle">
        <property name="text">
         <string>MqttState:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_MqttState">
        <property name="text">
         <string>MqttState</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_payload">
        <property name="text">
         <string>Payload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_topic">
        <property name="text">
         <string>Topic</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>Hello</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMqtt">
    <property name="title">
     <string>Mqtt</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <addaction name="menuHello"/>
   <addaction name="menuMqtt"/>
   <addaction name="menuSet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
